docs/bootstrap/digitalocean-terraform/main.tf

```diff
@@ -1,5 +1,11 @@
 terraform {
-  required_version = ">= 0.12"
+  required_version = "1.0.4"
```

Comment:
are we pinned to 1.0.4? id think we can allow any 1.X now Hashicorp has a commitment for 18months of 1.x compatibility

Reply:
np. I am used to pin it down, to have reproducable CI runs and no suprise pikachu :D